service/create/service.go

```diff
@@ -243,7 +239,7 @@ func (s *Service) Boot() {
 						policy = &awsresources.Policy{
 							ClusterID: cluster.Spec.Cluster.Cluster.ID,
 							KMSKeyArn: kmsKey.Arn(),
-							S3Bucket:  s.s3Bucket,
+							S3Bucket:  cluster.Name,
```

Comment:
I don't understand the problem. S3 buckets are like file system trees. They are separated by `/`. I only suggested to use another piece of information to structure the tree. The organization ID, aka customer ID, should always be there. The behaviour does not change because of the path structure of the bucket name. So there will still be buckets per cluster. It is then only that the cluster buckets are all grouped together in the organization bucket.